docs/Examples.md

```diff
@@ -413,6 +413,11 @@ including the `apps_location` parameter in your default settings. The value
 may either be a comma-separated list of apps or a YAML list, with each app
 referenced using a filesystem path or URL.
 
+Note: In the case of `SearchHeadCluster` or `ClusterMaster` when the apps are configured through 
+the `apps_location`, all those apps will be deployed to the Search Heads or Indexers respectively.
+In order to install the apps locally to the Deployer or ClusterMaster, the apps should be specified through
```

Comment:
To install the apps locally to the Deployer or ClusterMaster, the apps should be specified through `apps_location_local`.